libs/market-depth/src/lib/orderbook-row.tsx

```diff
@@ -89,42 +131,46 @@ export const OrderbookContinuousRow = React.memo(
     positionDecimalPlaces,
     indicativeVolume,
     price,
-    relativeAsk,
-    relativeBid,
     onClick,
+    type,
   }: OrderbookRowProps) => {
-    const type = bid ? 'bid' : 'ask';
     const value = bid || ask;
-    const relativeValue = bid ? relativeBid : relativeAsk;
     return (
-      <>
-        <VolCell
-          testId={`bid-ask-vol-${price}`}
-          value={value}
-          valueFormatted={addDecimalsFixedFormatNumber(
-            value,
-            positionDecimalPlaces
-          )}
-          relativeValue={relativeValue}
-          type={type}
+      <div className="relative w-full">
+        <CumulationBar
+          cumulativeRelativeAsk={cumulativeRelativeAsk}
+          cumulativeRelativeBid={cumulativeRelativeBid}
         />
-        <PriceCell
-          testId={`price-${price}`}
-          value={BigInt(price)}
-          onClick={() => onClick && onClick(addDecimal(price, decimalPlaces))}
-          valueFormatted={addDecimalsFixedFormatNumber(price, decimalPlaces)}
-        />
-        <CumulativeVol
-          testId={`cumulative-vol-${price}`}
-          positionDecimalPlaces={positionDecimalPlaces}
-          bid={cumulativeBid}
-          ask={cumulativeAsk}
-          relativeAsk={cumulativeRelativeAsk}
-          relativeBid={cumulativeRelativeBid}
-          indicativeVolume={indicativeVolume}
-        />
-      </>
+        <div className="grid gap-1 text-right auto-rows-[17px] grid-cols-3 w-full grid-rows-1">
```

Comment:
Lots of these classes are not required, we don't need to account for rows because this grid only has one row

Reply:
removed redundant classes